.buildkite/scripts/utils.sh

```diff
@@ -0,0 +1,80 @@
+# An opinionated approach to manage the Elatic Qualifier for the DRA in a Google Bucket
+# Instead of using the ELASTIC_QUALIFIER env variable.
+fetch_elastic_qualifier() {
```

Comment:
Could a project have different qualifier for the same branch?

Reply:
Yes!

That's only for the release process, aka Staging. The release team will ask teams to:
1) Create an `alpha1` release.
2) Then a new release with a different qualifier -> `RC1`, ... `RC2` finally, 
3) Finally, the release is approved and they will take `RC2` as the valid one so Release Team will ask us to run the DRA without a qualifier for. the same git `commit`.

We initially supported the env variable called `ELASTIC_QUALIFIER` but since this is gonna happen to a bunch of other GitHub repositories, @dliappis and myself came up with cenntralising the qualifier and branch in a Google Bucket. So we update the Google bucket and then the DRA Buildkite pipeline will use it.

Reply:
so the answer is no, two Elastic Stack projects cannot have a different qualifier for the same branch. When the qualifier for branch X changes to Q, all projects must use the qualifier Q for branch X. Am I right?

Reply:
Yes, temporarily. Let me clarify.

Currently, each product team triggers a new DRA with a new qualifier. Eventually, they will converge with the same qualifier per branch, either within a few hours or within one or two days at the most.

So far, this is trying to solve what you just described if all the Projects enrolled on the Unified Release use the same approach as here. Otherwise, it's a manual process and what you just said  it can happen